### PR TITLE
[QPG6100] App level cleanup for mDNS and latest PASE flow

### PR DESCRIPTION
**Pull request by tima-q**
#### Problem
Application code required update for recent developments.
- Setting up a PASE session was no longer possible without using 'OpenDefaultPairingWindow'
-- the handler for the PBKDFParamRequest is not registered otherwise
- Generic service publish can be removed in favor of the mDNS code in place

 #### Summary of Changes
* Using OpenDefaultPairingWindow in all example apps
* Remove deprecated test Service code
* Update Task stack sizes
* Set supported Admins to 4 iso 16

### CODE REVIEW

File: examples/platform/qpg6100/project_include/CHIPProjectConfig.h [hunk 1]
```diff
@@ -131,3 +131,14 @@
...
+
+/**
+ *  @def CHIP_CONFIG_MAX_DEVICE_ADMINS
+ *
+ *  @brief
+ *    Maximum number of administrators that can provision the device. Each admin
+ *    can provision the device with their unique operational credentials and manage
+ *    their access control lists.
+ */
+#define CHIP_CONFIG_MAX_DEVICE_ADMINS 1
```
Comment (woody-apple) [inline examples/platform/qpg6100/project_include/CHIPProjectConfig.h:143]: I thought the number we were looking at was 3-5?

**Review by woody-apple — CHANGES_REQUESTED** [review]

File: examples/platform/qpg6100/project_include/CHIPProjectConfig.h (hunk 1 above)
Comment (tima-q) [reply examples/platform/qpg6100/project_include/CHIPProjectConfig.h:143]: Is there a spec minimum we need to develop with at this point ?

Comment (tima-q) [reply examples/platform/qpg6100/project_include/CHIPProjectConfig.h]: After discussion on Slack - set to 4 - a minimum of 3 + 1 to rotate credentials.

**Review by woody-apple — APPROVED** [review]

**Comment by woody-apple** [comment]
@andy31415 @saurabhst @msandstedt ?

**Review by andy31415 — APPROVED** [review]

File: src/platform/qpg6100/CHIPDevicePlatformConfig.h [hunk 2]
```diff
@@ -51,11 +51,11 @@
 // ========== Platform-specific Configuration Overrides =========
 
 #ifndef CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE
-#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE (3 * 1024 + 256)
+#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE (4 * 1024)
```
Comment (msandstedt) [inline src/platform/qpg6100/CHIPDevicePlatformConfig.h:54]: Is there a relevance to 4 here?  Is this related to CHIP_CONFIG_MAX_DEVICE_ADMINS?  If so, should we perhaps have:

```suggestion
#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE (CHIP_CONFIG_MAX_DEVICE_ADMINS * 1024)
```

Else, why not just set the #define to 4096?  I'm not sure what value there is in showing the arithmetic if the reader doesn't know where it came from.

Comment (tima-q) [reply src/platform/qpg6100/CHIPDevicePlatformConfig.h:54]: The calculation is there for readability purposes - calculating in kB boundaries.
Currently we lack heap tracking support to fully fine-tune.

The ADMIN config is already to optimize the coming OpCreds addition in terms of flash and RAM buffers used per admin.

**Review by msandstedt — APPROVED** [review]
